api/opencl_runtime_layer.asciidoc

```diff
@@ -14690,6 +14691,24 @@ If using layered extension {cl_khr_command_buffer_mutable_dispatch_EXT},
 usage>>.
 ====
 
+Simultaneous use is defined using the _prerequisite_ terminology from the
+<<_execution_model, execution model>>, and is an optional feature for devices
+to support concurrent executions of a command-buffer. A command-buffer must
+be created with {CL_COMMAND_BUFFER_SIMULTANEOUS_USE_KHR} to avoid undefined
+behavior if a simultaneous use usage pattern occurs.
+
+[[simultaneous-use]]
+Simultaneous Use:: When a command-buffer is submitted for
+execution without a prerequisite on all the previous submissions of the same
+command-buffer which are not in the {CL_COMPLETE} state.
+
+An example of simultaneous use would be two submissions of the same
+command-buffer to a single out-of-order queue, without any events or barriers
+used to express a dependency between the two enqueue calls. Using a single
+in-order queue, events, or barriers to express dependencies between submissions
+of the same command-buffer would each be ways to avoid simultaneous use and are
+valid usage of command-buffers created without the
+{CL_COMMAND_BUFFER_SIMULTANEOUS_USE_KHR} flag.
```

Comment:
Is this normative or informative?  Consider:

```suggestion
[NOTE]
====
An example of simultaneous use would be two submissions of the same
command-buffer to a single out-of-order queue, without any events or barriers
used to express a dependency between the two enqueue calls. Using a single
in-order queue, events, or barriers to express dependencies between submissions
of the same command-buffer would each be ways to avoid simultaneous use and are
valid usage of command-buffers created without the
{CL_COMMAND_BUFFER_SIMULTANEOUS_USE_KHR} flag.
====
```